diff --git a/examples/rig/pusher/rig.py b/examples/rig/pusher/rig.py
index 20ad932..cead803 100644
--- a/examples/rig/pusher/rig.py
+++ b/examples/rig/pusher/rig.py
@@ -82,7 +82,7 @@ if __name__ == "__main__":
     run_experiment(
         grill_her_td3_full_experiment,
         exp_prefix='rlkit-pusher-rig-example',
-        mode='here_no_doodad',
+        mode='ec2',
         variant=variant,
         # use_gpu=True,  # Turn on if you have a GPU
     )
